### ISSUE TITLE
VS code Install issue "Unable to delete code.exe"

### ISSUE THREAD

**Issue by aayushkamal** [microsoft/vscode#231019]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.94.1
- OS Version: win10

During the upgrade of the application, the below error is thrown:
C:\Program Files\Microsoft VS Code\code.exe
![Image](https://github.com/user-attachments/assets/b8170617-854e-4435-b9f4-f529e83545d7)

I am using system installer and running it as admin.
can anyone please help?

**Comment by gjsjohnmurray** [comment]
@aayushkamal is this a duplicate of your  #230997?

**Comment by aayushkamal** [comment]
Yes, Just now I have checked I am getting code.exe error for every version while upgrading it to higher.
any idea?

**Comment by aayushkamal** [comment]
There is no process related to vs code in task manager but while uninstalling, It didn't remove these files.
![Image](https://github.com/user-attachments/assets/30e1bd53-b3f3-4be3-a6cf-35444b3a6220)
